tasks/options/jscs.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
-	all: [ "*.js", "performance/**/*.js" ]
-};
\ No newline at end of file
+	all: [ "*.js", "performance/**/*.js", "scss/variables/**/*.js" ],
+	options: {
+		fix: true
+	}
```

Comment:
yep that shouldn't be here, I actually forgot to remove it. Thanks for reminding.